xmipp

```diff
@@ -698,6 +698,95 @@ def runTests(testNames):
     testsToDo = '--all' if len(testNames)==0 else ' '.join(testNames)
     runJob("cd %s; %s base.py %s" % (testsPath, python, testsToDo))
 
+def readModelsFile():
+    with open(os.path.join("src", "xmipp", "models", "models.list"), 'r') as f:
+        lines = f.readlines()
+        return [line.strip('\n') for line in lines if not line.startswith('#')]
+
+def addDeepLearninModel(modelPath, update=False):
+    """ Takes the folder from modelPath and makes a .tgz,
+        uploads the .tgz to scipion web and
+        adds the modelName to the model.list.
+    """
+    if not os.path.isdir(modelPath):
+        print("modelPathDir must be a folder. \n"
+              "The name of the model will be that folder name.\n")
+        usage()
+        sys.exit(1)
+
+    modelName = os.path.basename(modelPath)
+    modelList = readModelsFile()
+    if modelName in modelList:
+        print(red("'%s' model name is already used. " % modelName))
+        if update:
+            print("Are you sure that you want to OVERWRITE that model: (yes/NO)")
+            yesno = raw_input()
+            if yesno.lower() != 'yes':
+                sys.exit(1)
+        else:
+            print("Please, choose one skipping the following list:")
```

Comment:
It will be better: print("Please, choose **a name avoiding one from** the following list:")